pkg/maps/srv6map/state.go

```diff
@@ -0,0 +1,122 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Authors of Cilium
+
+package srv6map
```

Comment:
If this map is going to be modified only from the datapath, why do we need to expose it to the control plane?

Reply:
This is only to have a couple helpers that are used by `cilium bpf srv6 state`.